src/pages/console/adminui/extend-cli.mdx

```diff
@@ -1,50 +1,90 @@
 
-import all1 from "/src/fragments/pull-cli-studio.mdx";
+import all1 from '/src/fragments/pull-cli-studio.mdx';
 
-<Fragments fragments={{all: all1}} />
+<Fragments fragments={{ all: all1 }} />
 
 ## Adding features with the Amplify CLI
 
-The Studio **Set up** menu lists cloud backend features such as functions, storage, and APIs that you can add to your app using the CLI. For each backend resource, Studio displays the CLI commands to run in your terminal window. 
+The Studio **Set up** menu lists cloud backend features such as functions, storage, and APIs that you can add to your app using the CLI. For each backend resource, Studio displays the CLI commands to run in your terminal window.
 
 1. In Studio, on the **Set up** menu, choose **Storage** or **Functions**.
-2. On the **Storage** page, under **Pull your Amplify project**, copy the following command and run it in your terminal window. 
+2. On the **Storage** page, under **Pull your Amplify project**, copy the following command and run it in your terminal window.
 3. After your backend environment has been successfully pulled from the cloud, copy the following command under **Add storage capabilities** and paste it in your terminal window:
+
 ```bash
 amplify add storage
 ```
+
 4. Follow the prompts in the terminal window to configure storage with your desired settings.
 5. Run the following command in your terminal window to build your local backend resources and provision them in the cloud:
+
 ```bash
 amplify push
 ```
+
 6. Return to the Studio **Storage** page. Confirm that a link to your new storage resource is available in the **Deployed storage resources** section.
 
 ## Infrastructure-as-code
 
 All backends created in Amplify Studio and Amplify CLI are deployed using AWS CloudFormation. AWS CloudFormation allows you store your backend configuration as code in your repository. This has three major benefits:
 
 1. Ability to extend a project with capabilities offered by individual services (e.g. Cognito, DynamoDB) that are not available in Amplify.
-2. Ability to easily replicate backend environments across AWS accounts and regions. 
-3. Ability to easily integrate with CI/CD pipelines such as the pipeline offered in the Amplify console. 
+2. Ability to easily replicate backend environments across AWS accounts and regions.
+3. Ability to easily integrate with CI/CD pipelines such as the pipeline offered in the Amplify console.
+
+All backend infrastructure-as-code definitions can be added to your project with the Amplify Command Line Interface (CLI).
+
+## Troubleshooting
+
+### Logging into Amplify Studio using Amplify CLI does not provide a Token or Prompt
+
+This behavior is observed when running a Amplify CLI `amplify pull --appId <appID> --envName <envName>` command in the terminal, which opens a new tab in the browser with a Amplify Studio login screen. On logging in, if you are not redirected back and observe the CLI being stuck in a loading state.
```

Comment:
This is a broken sentence

`On logging in, if you are not redirected back and observe the CLI being stuck in a loading state.`